libensemble/tests/regression_tests/test_mpi_runners.py

```diff
@@ -203,7 +209,8 @@ def run_tests(mpi_runner, runner_name, test_list_exargs, exp_list):
     H, pinfo, flag = libE(sim_specs, gen_specs, exit_criteria, persis_info, libE_specs=libE_specs)
 
 
-for run_set in ['mpich', 'openmpi', 'aprun', 'srun', 'jsrun', 'rename_mpich', 'custom']:
+# for run_set in ['mpich', 'openmpi', 'aprun', 'srun', 'jsrun', 'rename_mpich', 'custom']:
```

Comment:
Why drop openmpi?

Reply:
openmpi has to use a machinefile. So we cant test it using runline checks.